packages/muelu/doc/TutorialReadTheDocs/pages/quick_start.rst

```diff
@@ -0,0 +1,192 @@
+===========
+Quick start
+===========
+
+The first example is meant to quickly get into touch with MueLu.
+
+.. _quick_start/example problem:
+
+Example problem
+===============
+
+We generate a test matrix corresponding to the stencil of a 2D Laplacian operator on a structured Cartesian grid.
+The matrix stencil is
+
+.. math::
+   \frac{1}{h^2}\begin{pmatrix} & -1 & \\ -1 & 4 & -1 \\ & -1 & \end{pmatrix}
+
+where :math:`h` denotes the mesh size parameter.
+The resulting matrix is symmetric positive definite.
+We choose the right hand side to be the constant vector one
+and use a random initial guess for the iterative solution process.
+The problem domain is the unit square with a Cartesian (uniform) mesh.
+
+User interface
+====================
+
+For this tutorial there is an easy-to-use user interface to perform some experiments with multigrid methods for the given problem as described in :ref:`quick_start/example problem`. To use the user-interface run **./hands-on.py** in a terminal in the **../../../test/tutorial** folder.
+First one has to choose a problem. For this tutorial the right choice is the option 0 for the Laplace 2D problem on a :math:`50 \times 50` mesh.
+
+.. image:: pics/tut1_1.png
+  :width: 10cm
+
+Next one has to choose a xml file with the multigrid parameters. Choose option 2 and put in **s1_easy.xml** as filename for the xml file containing the xml parameters that are used for the multigrid method.
+
+.. note:
+
+  Please make sure that you enter a filename that actually exists on your hard disk!
+
+.. image:: pics/tut1_2.png
+  :width: 10cm
+
+The s1_easy.xml file has the following content
+
+.. literalinclude:: ../../../test/tutorial/s1_easy.xml
+  :language: xml
+
+As one can easily find from the xml parameters, a multigrid method with not more than 3 levels and a damped Jacobi method for level smoothing shall be used.
+Next, choose option 0 and run the example. That is, the linear system is created and iteratively solved both by a preconditioned CG method (from the AztecOO package) with a MueLu multigrid preconditioner and a standalone multigrid solver (again using MueLu) with the given multigrid parameters.
+
+.. image:: pics/tut1_3.png
+  :width: 10cm
+
+Note that the line **mpirun -np 2 MueLu_tutorial_laplace2d.exe -nx ...** is the command that is executed in the background. The default is 2 processors used.
+After pressing a key we are ready for a first analysis as it is stated by the green letters **Results up to date!**
```

Comment:
This talks about the green text with **Results up to date!**, but the image does not reflect it. It was a little confusing to see at first.